mappings/net/minecraft/client/particle/Particle.mapping

```diff
@@ -7,9 +7,9 @@ CLASS dik net/minecraft/client/particle/Particle
 	FIELD b prevPosX D
 	FIELD c prevPosY D
 	FIELD d prevPosZ D
```

Comment:
`prevX`, etc.

Reply:
I'm not sure I agree with posX -> x, etc.; Entity uses posX still, no?

Reply:
No, entity and most other thinfs use x y z.